### PR TITLE
handle empty bracket, w/ downstream errors

### PR DESCRIPTION
**Pull request by MishalAshraf**
w/ regards to issue I commented on previously!

### CODE REVIEW

**Comment by skekre98** [comment]
Please also see my comment on #145 regarding the null pointer check. Can you also show me a screenshot of what the playoff bracket looks like now?

File: modules/transformer.py [hunk 1]
```diff
@@ -96,17 +96,18 @@ def create_html_bracket(bracket_map):
...
-    if west == [[]] or east == [[]]:
-        playoff_map = {
-        "wc1" : [],
-        "wc2" : [],
-        "wcf" : [],
-        "ec1" : [],
-        "ec2" : [],
-        "ecf" : [],
-        "f" : []
+    if west == [[]] or east == [[]]: # if returned scraped data is empty, need to return an empty map with correct fields
```
Comment (skekre98) [inline modules/transformer.py:99]: You can just change this to `if not west[0] or not east[0]:` no need to create a new list for conditional.

File: modules/transformer.py [hunk 2]
```diff
@@ -96,17 +96,18 @@ def create_html_bracket(bracket_map):
...
+        "wc1" : [[["",""],["",""]], [["",""],["",""]], [["",""],["",""]], [["",""],["",""]]],
+        "wc2" : [[["",""],["",""]], [["",""],["",""]]],
+        "wcf" : [[["",""],["",""]]],
+        "ec1" : [[["",""],["",""]], [["",""],["",""]], [["",""],["",""]], [["",""],["",""]]],
+        "ec2" : [[["",""],["",""]], [["",""],["",""]]],
+        "ecf" : [[["",""],["",""]]],
+        "f" :   [[["",""],["",""]]]
         }
-        return playoff_map
+        print(empty_map)
```
Comment (skekre98) [inline modules/transformer.py:109]: Please remove this prior to merge.

File: modules/transformer.py [hunk 3]
```diff
@@ -94,6 +94,19 @@ def create_html_bracket(bracket_map):
     west = [bracket_map["Western Conference First Round"]]
     east = [bracket_map["Eastern Conference First Round"]]
     finals = []
+    playoff_map = {}
+
+    if not west[0] or not east[0]: # if returned scraped data is empty, need to return an empty map with correct fields
+        empty_map = {
+        "wc1" : [[["",""],["",""]], [["",""],["",""]], [["",""],["",""]], [["",""],["",""]]],
```
Comment (skekre98) [inline modules/transformer.py:101]: @MishalAshraf can you change the inner entries to tuples. Let's try to keep it in sync with the other entries in the `dict`. See line 131 as an example.

So `["",""],["",""]` should be `("",""),("","")` for all entries in the map

**Comment by skekre98** [comment]
Great work @MishalAshraf, one minor change and we should be good to merge 👍

**Review by skekre98 — APPROVED** [review]
LGTM 🎉

**Comment by MishalAshraf** [comment]
> LGTM 🎉

Awesome! Thanks for your patience, my first actual PR!

**Comment by skekre98** [comment]
> Awesome! Thanks for your patience, my first actual PR!

Congrats on your first successful open source commit 🎉

**Pull request merged**